### ISSUE TITLE
Reinitializate views

### ISSUE THREAD

**Issue by IZharov** [edvin/tornadofx#1107]
Hello,
I've created a tornado fx project that consists of several sequential views.
When I repeat the action again, the view is not redefined.
I want to reinitializate view when use replaceWith.
Views are on the same stage.
How to do it?

Part class RootView:
```
find(RootView::class).replaceWith<StartView>()
```
Part class StartView:
```
replaceWith<RootView>()
```
class TornadoApp:
```
class TornadoApp: App(StartView::class) {
    override fun start(stage: Stage) {
        stage.initStyle(StageStyle.UNDECORATED)
        super.start(stage)
    }
```

**Comment by ruckustboom** [comment]
You probably want to look into `Fragment`s

**Comment by ronsmits** [comment]
http://ronsmits.org/2017/01/tornadofxcdi/  this is a blog post I wrote some
time ago about this. Maybe it helps you understand the difference between
fragments and views

**Comment by edvin** [comment]
TL;DR: A View is a singleton within it's scope, so either extend `Fragment` - which will be a new instance every time, or open the `View` in a new scope. Understanding scopes is key to a well structured application (in many instances at least), so you should really head over to the guide and get these concepts down :)

**Comment by ctadlock** [comment]
Closing as answered
